src/js/themes/base.js

```diff
@@ -156,6 +156,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           color: 'border',
         },
       },
+      debounceTimer: 300, // The time to wait after the user stopped typing, measured in ms.
```

Comment:
This is very picky, but can we call it `debounceDelay`? I think I'm getting stuck on '300' not being a 'timer'.